RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -2203,8 +2218,25 @@ void setTrackCandidateBranches(LSTEvent* event,
                                                                                            trk_simhit_simTrkIdx,
                                                                                            trk_ph2_simHitIdx,
                                                                                            trk_pix_simHitIdx,
+                                                                                           percent_matched,
                                                                                            matchfrac);
 
+    int nPixHits = 0, nOtHits = 0, nLayers = 0;
+    for (int layerSlot = 0; layerSlot < Params_TC::kLayers; ++layerSlot) {
+      if (trackCandidatesExtended.lowerModuleIndices()[tc_idx][layerSlot] == lst::kTCEmptyLowerModule)
+        continue;
+      ++nLayers;
+      const bool isPixel = (trackCandidatesExtended.logicalLayers()[tc_idx][layerSlot] == 0);
+      if (isPixel)
+        nPixHits += 2;
+      else
+        nOtHits += 2;
```

Comment:
are these `2` the same as `kHitsPerLayer` ?
perhaps change.
Although perhaps there is a benefit to check that the hit index is not a sentinel